wallet/src/main/res/values/strings.xml

```diff
@@ -56,6 +56,10 @@
     <string name="provisioning_error">Error provisioning: %1$s</string>
     <string name="provisioning_unexpected">Unexpected state: %1$s</string>
 
+    <!-- Presentation screen -->
+    <string name="presentation_authkey_usage_warning">Using previously used Auth Key</string>
+    <string name="presentation_biometric_prompt_title">Authenticate to Share Your Credential</string>
```

Comment:
I think we should just say "Authentication Required"... it's shorter (space is at a premium) and it's already obvious that they are sharing their credential.